packages/main/src/StepInput.ts

```diff
@@ -106,6 +111,32 @@ const INITIAL_SPEED = 120; // milliseconds
  * @public
  */
 @event("change")
+/**
+ * Fired before the value state of the component is updated internally.
+ * The event is preventable, meaning that if it's default action is
+ * prevented, the component will not update the value state.
+ *
+ * @allowPreventDefault
+ * @public
+ * @param {string} valueState The new <code>valueState</code> that will be set.
+ * @param {boolean} valid Indicator if the value is in correct format pattern and in valid range.
```

Comment:
The documentation here says format-pattern.

Maybe we can change it to - Indicator if the value is correct and in range (if limitations are given with <code>min</code> or <code>max</code> properties.

Reply:
that's why we should not copy and paste documentation :D . Fixed.